src/Compilers/Core/Portable/MetadataReference/MetadataReferenceProperties.cs

```diff
@@ -134,6 +143,8 @@ public ImmutableArray<string> Aliases
         /// </summary>
         public bool EmbedInteropTypes => _embedInteropTypes;
 
+        internal bool IsRecursive { get; private set; }
```

Comment:
This property deserves a detailed comment. I am not sure I have a clear understanding what it means at this point.